src/g_game.c

```diff
@@ -546,6 +546,17 @@ void G_PrepGyroTiccmd(void)
   }
 }
 
+static boolean CheckDeathUseAction(void)
```

Comment:
Alright, I understand that a dead player cannot trigger the use button anymore if a non-default on-death-action is selected. But how does moving the logic to `P_DeathThink()` and setting `playerstate` there or load a game there maintain demo compatibility?

Reply:
The idea is that we don't write the `BT_USE` flag to the ticcmd `buttons`, so it won't be saved in the demo or sent over the network. If a dead player presses the "use" button, it won't register.

Reply:
I see. But it will still reload his last savegame if `death_use_action == death_use_reload`.

Reply:
You're right, I added check here: https://github.com/fabiangreffrath/woof/pull/1910/commits/eadfbaae4587f023502107d27d8fb439f6169691